packages/radix-ui-themes/src/styles/tokens.css

```diff
@@ -25,3 +25,16 @@
     --scaling: 1.1;
   }
 }
+
+/* cursors */
+.radix-themes {
+  --cursor-button: default;
+  --cursor-checkbox: default;
+  --cursor-disabled: not-allowed;
+  --cursor-link: pointer;
+  --cursor-menu-item: default;
+  --cursor-radio: default;
+  --cursor-slider-thumb: default;
+  --cursor-slider-thumb-active: default;
```

Comment:
So… you did the thing? 😀

Reply:
Well, the only thing that makes sense other than `cursor: default` on the slider thumb was `cursor: grab` and `cursor: grabbing` combo, otherwise makes no sense to have the variable giving you only half of what makes sense.